snippets/deepFreeze.md

```diff
@@ -6,13 +6,13 @@ tags: object,recursion,intermediate
 Deep freezes an object.
 
 Use `Object.keys()` to get all the properties of the passed object, `Array.prototype.forEach()` to iterate over them.
-Call `Object.freeze(obj)` recursively on all properties, checking if each one is frozen using `Object.isFrozen()` and applying `deepFreeze()` as necessary.
+Call `Object.freeze(obj)` recursively on all properties, checking if each object is frozen using `Object.isFrozen()` and applying `deepFreeze()` as necessary.
```

Comment:
Given the current situation in my country, I want to enter the global market internationally and have a bank account in foreign countries.